census/tftransformestimator/requirements.txt

```diff
@@ -0,0 +1,46 @@
+apache-beam==2.2.0
```

Comment:
Can we test if this works with the latest versions of all libraries?

Reply:
Hi David, 
moving to the new versions of all libraries require minor changes to requirement.txt and requirements_dataflow.txt. However, I would suggest to postpone them as the rest of the examples are still in tf 1.4.

Speak soon,
Lucio